tests/_sample-test.spec.ts

```diff
@@ -0,0 +1,15 @@
+import coverageData from "../data/coverage.json";
+import generateCMPTests from "../playwright/runner";
+
+const region = (process.env.REGION || "US").trim();
+
+Object.keys(coverageData).forEach((name) => {
+  if (!coverageData[name][region]) {
+    // no coverage in this region
+    return;
+  }
+  const siteList = coverageData[name][region].exampleSites.slice(0, 1);
```

Comment:
Should we pick more than one site? How long does it take to run with the current data?

Reply:
It's currently running ~100 tests per region. That takes 3-4 minutes on CI (https://jenkins.duckduckgo.com/job/cookie%20prompt%20management/job/autoconsent-ci/view/change-requests/job/PR-323/). I guess we could run more if you don't mind waiting >10 mins for results.